etc/ProdOfflineConfiguration.py

```diff
@@ -125,7 +125,8 @@
 alcaPPSScenario = "AlCaPPS_Run3"
 hiTestppScenario = "ppEra_Run3_pp_on_PbPb_2023"
 hiRawPrimeScenario = "ppEra_Run3_pp_on_PbPb_approxSiStripClusters_2023"
-hltScoutingScenario = "hltScoutingEra_Run3_2025"
```

Comment:
Hello,
at a first look this seems a mistake (was it done on purpose?) 
Is a revert of the change requested in https://github.com/dmwm/T0/issues/5069 and applied in https://github.com/dmwm/T0/pull/5072